README.md

```diff
@@ -245,7 +249,7 @@ aem {
     config {
         localInstance("http://localhost:4502") // local-author
         localInstance("http://localhost:4502") { // local-author
-            user = "admin"
+            user = "admin" //<- this won't compile: override val user: String = USER
```

Comment:
that's intentional. while we are creating instances from the scratch, we are just using admin account; it was more problematic than it looks like

Reply:
yes, that's OK, but this script won't compile, I would remove that line and comment, that admin is default